optimizers/prioritized.js

```diff
@@ -0,0 +1,363 @@
+var path = require("path");
```

Comment:
I do think this whole file should be a separate package actually that is installed and referenced.